docs/administration-guide/databases/mongodb.md

```diff
@@ -7,6 +7,13 @@ This article covers:
 - [Connecting to a MongoDB Atlas cluster](#connecting-to-a-mongodb-atlas-cluster).
 - [General connectivity concerns](#general-connectivity-concerns).
 
+## How does MongoDB work in Metabase
+Metabase automagically converts the selection you make in the UI into MongoDB syntax to see the data in a visual representation. However, structured data is easier to query than non-structure data due to its nature: with structured data you know the fields and types that a query will return while you can't be sure in the non-structured data unless you force schemas or types at the code or database level.
+
+As Metabase needs to know a few things before it can provide users with pre-populated filters or even the difference between a `datetime` or a `string` field, it will scan the first 200 documents in your collections to populate its metadata tables so it can give you the full power of the interface without overloading your MongoDB engine with a full collection scan.
+
+This technical decision is very important to understand in the case of any No-SQL database as it can have impact in the visualization of your questions and even problems with documents or fields not appearing in your results. For more information, check out our [troubleshooting guide](../../troubleshooting-guide/datawarehouse.html)
```

Comment:
```suggestion
Because MondoDB contains unstructured data, Metabase takes a different approach to syncing your database's Metadata. To get a sense of the schema, Metabase will scan the first 200 documents of each collection in your MongoDB. This sampling helps Metabase do things like differentiate datetime fields from string fields, and provide people with pre-populated filters. The reason Metabase only scans a sample of the documents is because scanning every document in every collection on every sync would be put too much strain on your database. And while the sampling does a pretty good job keeping Metabase up to date, it can also mean that new fields can sometimes fall through the cracks, leading to visualization issues, or even fields failing to appear in your results. For more information, check out our [troubleshooting guide](../../troubleshooting-guide/datawarehouse.html)
```

Reply:
s/MondoDB/MongoDB/
s/Metadata/metadata/